src/groups/circle_group.jl

```diff
@@ -24,9 +24,13 @@ end
 
 Base.show(io::IO, ::CircleGroup) = print(io, "CircleGroup()")
 
-adjoint_action(::CircleGroup, p, X) = X
+adjoint_action(::CircleGroup, p, X, ::LeftAction) = X
```

Comment:
Is it breaking that we no longer have the original signature `adjoint_action(::CircleGroup, p, X)`? Or is there a global default that fills in a `LectAction()`for that case (maybe I missed that then)?

Reply:
Yes, it is a global default. You can always use `adjoint_action(G, p, X)`, and this is redirected to `adjoint_action(G, p, X, LeftAction())`.